app/javascript/components/lesson-preview/index.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
 import {
   Tab, Tabs, TabList, TabPanel,
 } from 'react-tabs';
 import Prism from 'prismjs';
 
 import LessonContentInput from './components/lesson-content-input';
 import LessonContentPreview from './components/lesson-content-preview';
+import ShareButton from './components/share-button';
 import axios from '../../src/js/axiosWithCsrf';
 
-import { generateLink, encodeContent, decodeContent } from '../../src/js/previewShare';
-
-const LessonPreview = () => {
+const LessonPreview = ({previewContent}) => {
   const [content, setContent] = useState('');
```

Comment:
if I set the `content` with `previewContent` by default it returns the following error:

```
Warning: `value` prop on `textarea` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
```
Am I doing something wrong?

Reply:
It will be the `@preview.content` on this line causing that. A new record will have nil content.
```ruby
<%= react_component('lesson-preview/index', { previewContent: @preview.content }) %>`
```

But we can call `to_s` on it to ensure we always get a string, even when content is nil.
```ruby
<%= react_component('lesson-preview/index', { previewContent: @preview.content.to_s }) %>
```

Reply:
Yep I noticed that I fixed it already 

Thank you!